### PR TITLE
docs: update rpk container examples

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves https://github.com/redpanda-data/documentation-private/issues/938

### CODE REVIEW

**Review by r-vasquez — APPROVED** [review]
Thanks!

File: modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc [hunk 1]
```diff
@@ -11,6 +11,32 @@ Start a local container cluster.
 rpk container start [flags]
 ----
 
+== Examples
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc]:
```suggestion
== Example
```

File: modules/reference/pages/rpk/rpk-container/rpk-container-status.adoc [hunk 2]
```diff
@@ -10,6 +10,29 @@ Get a local container's status.
 rpk container status [flags]
 ----
 
+== Examples
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-container/rpk-container-status.adoc]:
```suggestion
== Example
```

**Comment by micheleRP** [comment]
@Deflaimun: What is `uint` in `rpk container start -n` and `--retries`? Typo?

File: modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc [hunk 3]
```diff
@@ -11,6 +11,32 @@ Start a local container cluster.
...
+
+If you're following the guide xref:get-started:quick-start.adoc[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container start -n 3` to start your containers:
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc]:
```suggestion
If you're following the xref:get-started:quick-start.adoc#deploy-redpanda[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container start -n 3` to start your containers:
```

File: modules/reference/pages/rpk/rpk-container/rpk-container-status.adoc [hunk 4]
```diff
@@ -10,6 +10,29 @@ Get a local container's status.
...
+
+If you're following the guide xref:get-started:quick-start.adoc[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container status` to see more information about your containers:
```
Comment (micheleRP) [inline modules/reference/pages/rpk/rpk-container/rpk-container-status.adoc]:
```suggestion
If you're following the xref:get-started:quick-start.adoc#deploy-redpanda[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container status` to see more information about your containers:
```

@Deflaimun: I wanted to link straight to the 3 broker steps, but couldn't get this to work xref:get-started:quick-start.adoc?tab=tabs-1-three-brokers. Are we able to link into tabs?

**Review by micheleRP — APPROVED** [review]
approving with some requested edits

File: modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc (hunk 3 above)
Comment (Deflaimun) [reply modules/reference/pages/rpk/rpk-container/rpk-container-start.adoc]: Checked with Jake how to do it. We need to put #id-of-tab. So in this case it's `xref:get-started:quick-start.adoc#tabs-1-three-brokers`

**Pull request merged**